mission_control/src/behaviors/attitude_servo.cpp

```diff
@@ -97,7 +93,7 @@ BT::NodeStatus AttitudeServoBehavior::behaviorRunningProcess()
   else
   {
     ros::Duration delta_t = ros::Time::now() - behaviorStartTime_;
-    if (delta_t.toSec() > timeOut_)
+    if (delta_t.toSec() > timeOut_ && timeOutEnable_)
```

Comment:
@patapufate nit: consider checking the flag before computing and comparing durations.

Reply:
Fixed